coala_quickstart/info_extraction/InfoScope.py

```diff
@@ -0,0 +1,91 @@
+from coala_quickstart.info_extraction.Utilities import assert_type_signature
+from coala_quickstart.info_extraction.InfoExtractor import InfoExtractor
+
+
+class InfoScope:
+
+    def __init__(self,
+                 level,
+                 sections=[],
+                 bears=[],
+                 allowed_sources=[],
+                 allowed_extractors=tuple()):
+        """
+        A class representing the scope of applicability of an
+        ``Info`` instance. An ``InfoScope`` instance can be used to
+        restrict the usability of an ``Info`` instance as a setting
+        value.
+
```

Comment:
Shift this docstring before `__init__` as its related to the class and not this function.